src/data/events.ts

```diff
@@ -53,35 +57,49 @@ export async function getNext3Events(): Promise<[Event, Event, Event]> {
       },
     ];
   }
-  const next3Events: [Event, Event, Event] = [
-    {
-      title: 'Forschungsraum: Körper & Sexualität',
-      date: new Date(2024, 4, 17),
-      description: 'Ein Workshop mit Raum zum Austausch über Sexualität und unsere Körper.',
-      link: '/events/forschungsraum-korper',
-      image: {
-        src: image1,
-      },
-    },
-    {
-      title: 'Forschungsraum: Wutkraft',
-      date: new Date(2024, 5, 27),
-      description: 'Lasst uns die Kraft, die in unserer Wut steckt, gemeinsam erforschen.',
-      image: {
-        src: image3,
-      },
-    },
-    {
-      title: 'Sommer Gathering',
-      date: new Date(2024, 8, 2),
-      description: 'Zusammen werden wir erkunden, was Community für uns sein kann',
-      image: {
-        src: image2,
-      },
+  const today = new Date();
+  today.setHours(0, 0, 0, 0);
+  const response = await fetch(
+    `${process.env.CMS_URL}/api/events?sort=date&where[start][greater_than]=${today.toISOString()}&limit=3`,
```

Comment:
Maybe use [`qs`](https://www.npmjs.com/package/qs) for these query strings for better readability?